lib/rex/post/meterpreter/ui/console/command_dispatcher.rb

```diff
@@ -50,6 +50,23 @@ def client
   end
 
   #
+  # Returns the commands that meet the requirements
+  #
+  def check_commands(all, reqs=nil)
+    all.delete_if do |cmd, _desc|
+      del = false
+      reqs[cmd].each do |req|
+        next if client.commands.include? req
+        del = true
+        break
+      end
+      del
+    end
+
+    all
+  end
```

Comment:
Seems a little verbose compared to the original implementation of
``` ruby
    all.delete_if do |cmd, _desc|
      reqs[cmd].any? { |req| !client.commands.include?(req) }
    end
```

+1 for the refactor though, would be good to see this applied to the other plugins as well, though that's probably beyond the scope of this PR.